### PR TITLE
fixes nav dups in the main install page

### PR DESCRIPTION
**Pull request by desteves**
Moves and consolidates headers up a level so that these are not nested within the language choosable shortcode. This then prevents the right nav from rendering duplicates. Headers level 1 through level 3 are updated.

- Affected page(s): https://www.pulumi.com/docs/install/
- Fixes 3583

### CODE REVIEW

**Comment by desteves** [comment]
> http://pulumi-hugo-origin-pr-3625-7d48ef2f.s3-website.us-west-2.amazonaws.com

http://pulumi-hugo-origin-pr-3625-7d48ef2f.s3-website.us-west-2.amazonaws.com/docs/install/

**Comment by desteves** [comment]
@sean1588 I created this PR to get the site preview as suggested.

File: themes/default/content/docs/install/_index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title_tag: "Download & Install Pulumi"
 meta_desc: This page contains detailed instructions for downloading and installing Pulumi.
-title: "Download & install"
-h1: Download & install Pulumi
+title: "Download & Install"
+h1: Download & Install Pulumi
```
Comment (sean1588) [inline themes/default/content/docs/install/_index.md:5]: Any reason why we are adjusting the casing here? I believe we generally do sentence casing as opposed to title casing for our headings unless it is a proper noun. Though it does seem like there are some inconsistencies across the site with regards to this.

**Review by sean1588 — APPROVED** [review]

File: themes/default/content/docs/install/_index.md (hunk 1 above)
Comment (desteves) [reply themes/default/content/docs/install/_index.md:5]: Gotcha, I wasn't aware of a style around this. Happy to adhere to the sentence casing.

**Pull request merged**